utils/bazel/llvm-project-overlay/llvm/config.bzl

```diff
@@ -96,15 +82,7 @@ builtin_thread_pointer = select({
 })
 
 # TODO: We should split out host vs. target here.
```

Comment:
Note that I believe that this TODO can be removed, because this CL should fix it

I left it in, just in case it was also referring to something else here